src/odemis/dataio/hdf5.py

```diff
@@ -19,21 +19,20 @@
 You should have received a copy of the GNU General Public License along with 
 Odemis. If not, see http://www.gnu.org/licenses/.
 '''
-from past.builtins import basestring, unicode
-from numpy.polynomial import polynomial
-
 from collections.abc import Iterable
-import h5py
 import json
 import logging
-import numpy
-from odemis import model
-import odemis
-from odemis.util import spectrum, img, fluo
-from odemis.util.conversion import JsonExtraEncoder
 import os
 import time
 
+from numpy.polynomial import polynomial
+import h5py
+import numpy
+
+from odemis import model
+from odemis.util import fluo, img, spectrum
+from odemis.util.conversion import JsonExtraEncoder
+import odemis
```

Comment:
`import odemis` should go above `from odemis import model` according to our new coding standards.

Reply:
I thought we had to separate "from"s and "import"s.... but apparently not. So I'll fix it!